index.js

```diff
@@ -21,6 +22,8 @@ app.use(bodyParser.json())
 // port
 const port = 3002
```

Comment:
@julia-ediamond I'm genuinely curious, what's the benefit of having port 3002 over 3000?

Reply:
They are same. The reason why I switched to 3002, is that ports are like postal addresses, there can't be two similar. When I was installing Postgres before the class with Harry, I didn't know if I should keep default port 5432 or not. As 3000 seemed more familiar I changed it to 3000 in Postgres. Afterwards I got my project not working because it was on port 3000. I do need to change settings in Postgress and switch it back to 5432.